src/win/util.c

```diff
@@ -1388,7 +1388,7 @@ int uv__getpwuid_r(uv_passwd_t* pwd) {
   if (OpenProcessToken(GetCurrentProcess(), TOKEN_READ, &token) == 0)
     return uv_translate_sys_error(GetLastError());
 
-  bufsize = sizeof(path);
+  bufsize = sizeof(path) / sizeof(wchar_t);
```

Comment:
It's a minor thing but it would be better to use `ARRAY_SIZE(path)` here and below.